server/components/auth/redirectUrl.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by sourabh on 27/7/17.
+ */
+const EmployeeSchema = require("../../api/employee/employee.model");
+const jwt_token = require("jsonwebtoken");
+const Network = require("../network");
+const redirectUrl = (req, res) => {
+
+  const hrmsToken = req.query.access_token;
+  Network({url: "http://newers-world-oauth.qa2.tothenew.net/oauth/user?access_token=" + hrmsToken, method: 'GET'})
+    .then(employeeData => {
+      if (employeeData.statusCode === 200) {
+        const employee = JSON.parse(employeeData.body);
+        const employeeDetails = {
+          employeeEmail: employee.email,
+          employeeId: employee.employeeCode,
+        };
+        process.env.SECRET_KEY = 'NothingIsImpossible';
```

Comment:
Bad assignment.
SECRET_KEY either should come from process.env or have default value.
const SECRET_KEY = process.env.SECRET_KEY || 'default key value';